### ISSUE TITLE
Stakeholder research and engagement - Review research to date

### ISSUE THREAD

**Issue by lukegonis** [BCDevOps/developer-experience#1926]
**Describe the issue**
Review any interviews, workshops, surveys, or any other activities done with the internal stakeholders for Private Cloud. 

**Additional context**
These are the resource we have to date:
Background from the Competition Notice (put into the comments in this issue)
[Roadmap](https://app.mural.co/t/platformservices5977/m/cloudpathfinder2329/1623864264564/0527296c93e73799c641f7d9fae03f0139002aee?sender=udb0eaba57bd3b9a2fa8e3916 )
[Cloud IA](https://app.mural.co/t/platformservices5977/m/platformservices5977/1626806093438/ca4fb81b38837ee25cb5d502afde73e0607fbb0f?sender=udb0eaba57bd3b9a2fa8e3916)
[Service Overview](https://docs.google.com/presentation/d/1ItipubvqCR1rI_VVVbx_swX659HPgN_tiwiKJp1eL94/edit#slide=id.gbdc1011b6a_0_517)
[Executive Sprint Overview](https://docs.google.com/presentation/d/1Az-vZ9wJocqPpNTWpfsfcB4ipqKgvTXDfAxqiDaSIyM/edit#slide=id.gddc6dce5bd_0_0)
[Evidence Based decision making in driving devops transformation](https://docs.google.com/presentation/d/1u8EuIhcUU3OW0sdbxGaRoMAxkhj-tbHQkBtz_cUupn4/edit#slide=id.gddc6dce5bd_0_0)
[Private Cloud BMC](https://app.mural.co/t/platformservices5977/m/cloudpathfinder2329/1616704079748/fe26c5ca20ed0cdbc651f60b2370db708d34a9f0?sender=uf88144ee66b054ae5c871382)

**Definition of done**
- [x] The above documents have been reviewed and notes taken in a document, spreadsheet or Mural
- [x] List of outstanding questions created

**Comment by theresawuzhere** [comment]
This is the "Background" from the Competition Notice: 
The DevOps and Cloud Services Branch supports the Province’s Cloud Platform as a Service (PaaS) programs,
those being the established Private Cloud OpenShift solution and the evolving Public Cloud Pathfinder.
Service design research completed at the beginning of the 2021 fiscal year highlighted top priorities for the
evolution of the Cloud PaaS offerings. From the Private Cloud research, of four key themes identified, this
statement of work focuses on the theme, ‘making the service easier to use and understand.’ Specifically, our
three primary personas of developers, product owners and security specialists, are requesting a simplified web
presence. We are committed to consolidating and simplifying our existing websites and online resources to
make it easy to find and understand information about our service. In addition, this should lead to easier
access to improved technical documentation, improving service onboarding, and simpler wayfinding to
enhanced training materials. Primary outcomes of this work will lead to developers having access to the
resources available to learn how to build and deploy in Cloud PaaS services, and product owners able to
understand the services provided and how to leverage them.
DevOps and Cloud Services is looking for a Content Strategist to provide leadership and expertise in user
experience (UX) research, content strategy and information architecture. Using UX practices including research
through stakeholder interviews and process mapping, information architecture, search engine optimization,
web content development (wire framing and writing), and accessibility best practices and standards, the
Content Strategist will ensure that the DevOps and Cloud Services Branch’s online presence is user-focused and
aligned with corporate strategies and standards.
We are looking for people who have strong facilitation and plain language skills. Successful candidates will be
reporting to and working closely with the DevOps and Cloud Services Branch Private Cloud Platform Services
Team. For more information on the Government of B.C.’s writing and accessibility standards, please see
digital.gov.bc.ca/standards-and-guides.

**Comment by theresawuzhere** [comment]
This is the [WIP document](https://docs.google.com/document/d/1KZHh_Dpu34SrCGBOl-9LvLvcLYgoMTAzcCZafVzSUuo/edit?usp=sharing)

**Comment by theresawuzhere** [comment]
I've reviewed the relevant stakeholder documents and I have made a list of questions. Not all questions will go into the stakeholder interviews. We can answer some before the interviews. We will answer these questions and put the remaining questions into the interviews. 

Overall, I think that I'm not seeing a lot of specificity around WHO the content is for and at what STAGE of the user journey. I'm also not seeing a strategy around HOW information is sent to these different users. I still have to look at the user research and the user journey, and hope to uncover more information. 

All of these can be answered in a content strategy, and then the website and other channels can support the strategy.
